sdk/core/azure-core/azure/core/serialization.py

```diff
@@ -0,0 +1,19 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+__all__ = ["NULL"]
+
+class _Null(object):
+    """To create a Falsy object
+    """
+    def __bool__(self):
+        return False
+
+    __nonzero__ = __bool__ # Python2 compatibility
+
+
+NULL = _Null()
```

Comment:
We should have a (good) comment for this constant (that spinx picks up). Unless we have customized it away, a docstring:

```python
NULL = _Null()
"""Something something something
"""
```

should be picked up by sphinx.

Reply:
verified the documentation - and updated the toc tree